catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -120,10 +120,10 @@ public EventProcessorImpl(
    */
   public static void processEntry(Metacard metacard, String operation, EventAdmin eventAdmin) {
     String methodName = "processEntry";
-    LOGGER.debug("ENTERING: " + methodName);
+    LOGGER.debug("ENTERING: {}", methodName);
```

Comment:
Since we are touching these entry/exit logs, shouldn't they be at the trace level?